src/Illuminate/Auth/Notifications/VerifyEmail.php

```diff
@@ -59,7 +59,7 @@ public function toMail($notifiable)
     protected function verificationUrl($notifiable)
     {
         return URL::temporarySignedRoute(
-            'verification.verify', Carbon::now()->addMinutes(60), ['id' => $notifiable->getKey()]
+            'verification.verify', Carbon::now()->addMinutes(config('auth.verification.expire', 60)), ['id' => $notifiable->getKey()]
```

Comment:
I believe using `config()` here is not supposed to happen because `illuminate/auth` is also available as a standalone packge which does not depend on the frameworks global helpers.

_However:_ I see that in the `Auth` namespace we already use other global helpers in the code (*not* talking about the stubs), e.g:
- `url()`
- `route()`

I believe they shouldn't be here too...

Reply:
@mfn I don't think you can use illuminate/auth as a standalone package?

Reply:
In which case I totally have no objection!

https://github.com/illuminate/auth also features no readme or anything that it would be usable standalone.

Apologies for the confusion then!

Reply:
See #27050.